pages/index.html

```diff
@@ -4,9 +4,49 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Stan for Bidet</title>
+  <link href="/styles/style.css" rel="stylesheet"> 
+  <link href="/styles/index.css" rel="stylesheet"> 
+
 </head>
+
+<header>  
+  <nav class="top-banner">
+      <a href="index.html">Homepage</a>
+      <a href="gallery.html">Portraits of Bidets</a>
+      <a href="facts.html">Everything You Need To Know About Bidets</a>
+  </nav>
+</header>
```

Comment:
[nit] Make the spacing consistent. 😄 

```suggestion
<header>  
  <nav class="top-banner">
    <a href="index.html">Homepage</a>
    <a href="gallery.html">Portraits of Bidets</a>
    <a href="facts.html">Everything You Need To Know About Bidets</a>
  </nav>
</header>
```